etc/kayobe/environments/ci-multinode/stackhpc-ci.yml

```diff
@@ -51,8 +51,8 @@ stackhpc_repo_centos_stream_9_docker_version: "{{ stackhpc_pulp_repo_centos_stre
 stackhpc_repo_rhel_9_treasuredata_4_version: "{{ stackhpc_pulp_repo_rhel_9_treasuredata_4_version }}"
 stackhpc_repo_rhel_9_mariadb_10_6_version: "{{ stackhpc_pulp_repo_rhel_9_mariadb_10_6_version }}"
 stackhpc_repo_rhel_9_influxdb_version: "{{ stackhpc_pulp_repo_rhel_9_influxdb_version }}"
-stackhpc_repo_rocky_9_crb_version: "{{ stackhpc_pulp_repo_rocky_9_1_crb_version }}"
-stackhpc_repo_rocky_9_highavailability_version: "{{ stackhpc_pulp_repo_rocky_9_1_highavailability_version }}"
+stackhpc_repo_rocky_9_crb_version: "{{ stackhpc_pulp_repo_rocky_9_crb_version }}"
```

Comment:
I've just noticed I should have moved these lower down below the comment. I'll update them once the sync is finished, which will conveniently re-run CI